### ISSUE TITLE
serviceaccounts "service-catalog-controller-manager" already exists

### ISSUE THREAD

**Issue by schlichtanders** [aws-quickstart/quickstart-amazon-eks#138]
I am deploying an older version of the example workload of this quickstart https://github.com/aws-quickstart/quickstart-amazon-eks/blob/37f87d8472c55d26302a0174abf819bc65ee4da2/templates/example-workload.template.yaml

it works all well the first time, however the second time I get an error similar to this:
``` 
eks-example-workload HelmExample Custom::Helm CREATE_FAILED Failed to create resource. ERROR: (truncated) 07 13:00:13 failed install perform step: release maudlin-dog failed: serviceaccounts "service-catalog-controller-manager" already exists
```

indicating that the delete is not properly working. I was thinking, deleting should work, but am not sure - should it work?
Any help is highly appreciated

----------

How could I delete manually all extra resources created by the example-workload.template.yaml?

**Comment by jaymccon** [comment]
The delete stack action simply calls `helm uninstall <>`. If there are resources left behind by the chart, the cfn custom resource makes no additional efforts to clean them up.

**Comment by schlichtanders** [comment]
Thanks a lot for pointing out that such a cleanup would be rather simple.
Of course it would be great, if the correct cleanup is also included into the quickstart-eks eks-example-workload

Which concrete commands would this be for this eks-example-workload?

**Comment by jaymccon** [comment]
I think I could have done a better job on being clear in my previous comment, let me try again.

The Helm support for CloudFormation provided in this project wraps the helm client. CloudFormation stack `Create`/Update`/`Delete` operations are the equivalent of calling the helm cli with `helm ``install`/`upgrade`/`uninstall`. The wrapper has no knowledge of what components are in a given helm chart, and whether the helm client, or chart, performs a given operation as expected. So it would actually be not simple, but very hard, for this project to try to handle cleanup where the helm client have not done this correctly. 

That said, we don't want to have a broken example. I've updated the example to point to the grafana chart, which cleans up after itself properly in my tests.
